name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/ibflex/IBFlexStatementExtractor.java

```diff
@@ -710,20 +731,18 @@ private Unit createUnit(Element element, Unit.Type unitType, Money amount)
             }
             else
             {
-                // Calculate the FX rate to the base currency
-                BigDecimal fxRateToBase = element.getAttribute("fxRateToBase").isEmpty() ? BigDecimal.ONE
-                                : asExchangeRate(element.getAttribute("fxRateToBase"));
+                BigDecimal exchangeRate = getExchangeRate(element, amount.getCurrencyCode(), accountCurrency);
```

Comment:
I believe getExchangeRate can return null.

As I understand it, getExchangeRate is also checking for fxRateToBase *and* it now attempts to calculate the cross rate using the account currency, to probably it is not a change. But maybe a check for null could make sense